### PR TITLE
[T5A1][F09-B2]Antonia Devina

### PR DESCRIPTION
**Pull request by antoniadevina**

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -46,7 +46,10 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
+`* *` | user | edit contact details | change the details of a person
```
Comment (chanjunweimy) [inline doc/DeveloperGuide.md]: change what details? :)

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -46,7 +46,10 @@ Priority | As a ... | I want to ... | So that I can...
...
+`* *` | user | list persons with a particular tag | easily see who belong to the tag/group
```
Comment (chanjunweimy) [inline doc/DeveloperGuide.md]: what is the benefit of easily see who belong to the tag?

File: doc/DeveloperGuide.md [hunk 3]
```diff
@@ -46,7 +46,10 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
+`* *` | user | edit contact details | change the details of a person
+`* *` | user | list persons with a particular tag | easily see who belong to the tag/group 
 `*` | user with many persons in the address book | sort persons by name | locate a person easily
+`*` | user | seperate last name and first name | confinient for the user
```
Comment (chanjunweimy) [inline doc/DeveloperGuide.md]: confinient ? Do you mean convenient? However, the benefit is not precise enough... What do you mean by convenient?

File: doc/DeveloperGuide.md [hunk 4]
```diff
@@ -74,12 +77,42 @@ Use case ends.
...
+
+**MS** 
+
+1. User requests to rename tag 
+2. User enters the existing tag name that want to rename 
+3. User enters the new tag to replace the old tag 
+4. AddressBook asks for confirmation 
+5. User comfirms rename 
+6. AddressBook rename all the tags 
+Use case ends.
```
Comment (chanjunweimy) [inline doc/DeveloperGuide.md]: Use case should be in a "back and forth" manner, you can take a look at #534 for a sample. :) 
1) User
2) Addressbook
3) User 
4) Addressbook
..
is a correct workflow.

**Comment by jeffryhartanto** [comment]
@antoniadevina Putting wrong PR title may let other tutor to do more work. Next time pay attention to the name for the PR title. :)

@chanjunweimy Thanks for reviewing her PR. Sorry for the inconvenience.

**Pull request closed without merging**